Dockerfile

```diff
@@ -57,4 +57,6 @@ RUN mkdir -p "$(helm home)/plugins" && \
 # NOTE: Silent parallel
 RUN echo "will cite\n" | parallel --bibtex
 
+COPY scripts/ /scripts/
```

Comment:
Let's copy to /usr/local/bin so we don't have to fully qualify path.

Reply:
I'm ok with specifying the full path :) /usr/local/bin is for suckers.